docs/noir_js/getting_started/01_setup.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Setup
+description:
+  Learn how to setup a new app that uses Noir to generate and verify zero-knowledge SNARK proofs in a typescript or javascript environment
+keywords: [how to, guide, javascript, typescript, noir, barretenberg, zero-knowledge, proofs]
+---
+
+Noir JS works both on the browser and on the server, and works for both ESM and CJS module systems. In this page, we will learn how can we write a simple test and a simple webapp to verify the Standard Noir Example.
+
+## Before we start
+
+Make sure you have Node installed on your machine by opening a terminal and executing `node --version`. If you don't see a version, you should install [node](https://nodejs.org/en/download). You can also use `yarn` if you prefer that package manager.
```

Comment:
Is this your preferred way to install node? I usually use nvm, https://github.com/nvm-sh/nvm

Reply:
we could also suggest bun :grin: